curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-dot-notation-to-access-the-properties-of-an-object.english.md

```diff
@@ -21,10 +21,10 @@ Print both <code>properties</code> of the <code>dog</code> object below to your
 
 ```yml
 tests:
-  - text: Your should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.
-    testString: assert(/console.log\(.*dog\.name.*\)/g.test(code), 'Your should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.');
-  - text: Your should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.
-    testString: assert(/console.log\(.*dog\.numLegs.*\)/g.test(code), 'Your should use <code>console.log</code> to print the value for the <code>numLegs</code> property of the <code>dog</code> object.');
+  - text: Your code should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.
```

Comment:
I think the original intent was `Your` was supposed to be `You`, so there would be no need to add the word `code` here. 
```suggestion
  - text: You should use <code>console.log</code> to print the value for the <code>name</code> property of the <code>dog</code> object.
```